<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #20004b;
    font-size:20px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.1;
    font-weight: 800;
    border-radius: 10px;
    background-position: center center;
    background-size: cover;
    background-repeat: no-repeat;
    border-color: transparent;
     border-style: solid;
   display: table-cell;
    vertical-align: middle;
    width: 100%;
    -webkit-text-size-adjust: 100%;
border-radius: 10px;
    background-color: #ffde00;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
}</string>
   </property>
   <property name="text">
    <string>                                              Результаты теста</string>
   </property>
  </widget>
  <widget class="QLabel" name="persona">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #20004b;
    font-size:20px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.1;
    font-weight: 800;
    border-radius: 10px;
    background-position: center center;
    background-size: cover;
    background-repeat: no-repeat;
    border-color: transparent;
     border-style: solid;
   display: table-cell;
    vertical-align: middle;
    width: 100%;
    -webkit-text-size-adjust: 100%;
border-radius: 10px;
    background-color: #ffde00;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
}</string>
   </property>
   <property name="text">
    <string>ну типа тут</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: #ffffff;
    font-size: 14px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.55;
    font-weight: 700;
    border-radius: 10px;
    background-color: #ad26ff;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;

}</string>
   </property>
   <property name="text">
    <string>      Типы Личности</string>
   </property>
  </widget>
  <widget class="QLabel" name="persona2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #20004b;
    font-size:20px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.1;
    font-weight: 800;
    border-radius: 10px;
    background-position: center center;
    background-size: cover;
    background-repeat: no-repeat;
    border-color: transparent;
     border-style: solid;
   display: table-cell;
    vertical-align: middle;
    width: 100%;
    -webkit-text-size-adjust: 100%;
border-radius: 10px;
    background-color: #ffde00;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
}</string>
   </property>
   <property name="text">
    <string>ну типа тут</string>
   </property>
  </widget>
  <widget class="QLabel" name="persona3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #20004b;
    font-size:20px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.1;
    font-weight: 800;
    border-radius: 10px;
    background-position: center center;
    background-size: cover;
    background-repeat: no-repeat;
    border-color: transparent;
     border-style: solid;
   display: table-cell;
    vertical-align: middle;
    width: 100%;
    -webkit-text-size-adjust: 100%;
border-radius: 10px;
    background-color: #ffde00;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
}</string>
   </property>
   <property name="text">
    <string>ну типа тут</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: #ffffff;
    font-size: 14px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.55;
    font-weight: 700;
    border-radius: 10px;
    background-color: #ad26ff;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;

}</string>
   </property>
   <property name="text">
    <string>1 место</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: #ffffff;
    font-size: 14px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.55;
    font-weight: 700;
    border-radius: 10px;
    background-color: #ad26ff;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;

}</string>
   </property>
   <property name="text">
    <string>2 место</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: #ffffff;
    font-size: 14px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.55;
    font-weight: 700;
    border-radius: 10px;
    background-color: #ad26ff;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;

}</string>
   </property>
   <property name="text">
    <string>3 место</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: #ffffff;
    font-size: 14px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.55;
    font-weight: 700;
    border-radius: 10px;
    background-color: #ad26ff;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;

}</string>
   </property>
   <property name="text">
    <string>     Типы Интелекта</string>
   </property>
  </widget>
  <widget class="QLabel" name="iq">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #20004b;
    font-size:20px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.1;
    font-weight: 800;
    border-radius: 10px;
    background-position: center center;
    background-size: cover;
    background-repeat: no-repeat;
    border-color: transparent;
     border-style: solid;
   display: table-cell;
    vertical-align: middle;
    width: 100%;
    -webkit-text-size-adjust: 100%;
border-radius: 10px;
    background-color: #ffde00;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
}</string>
   </property>
   <property name="text">
    <string>ну типа тут</string>
   </property>
  </widget>
  <widget class="QLabel" name="iq2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #20004b;
    font-size:20px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.1;
    font-weight: 800;
    border-radius: 10px;
    background-position: center center;
    background-size: cover;
    background-repeat: no-repeat;
    border-color: transparent;
     border-style: solid;
   display: table-cell;
    vertical-align: middle;
    width: 100%;
    -webkit-text-size-adjust: 100%;
border-radius: 10px;
    background-color: #ffde00;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
}</string>
   </property>
   <property name="text">
    <string>ну типа тут</string>
   </property>
  </widget>
  <widget class="QLabel" name="iq3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #20004b;
    font-size:20px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.1;
    font-weight: 800;
    border-radius: 10px;
    background-position: center center;
    background-size: cover;
    background-repeat: no-repeat;
    border-color: transparent;
     border-style: solid;
   display: table-cell;
    vertical-align: middle;
    width: 100%;
    -webkit-text-size-adjust: 100%;
border-radius: 10px;
    background-color: #ffde00;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;
}</string>
   </property>
   <property name="text">
    <string>ну типа тут</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: #ffffff;
    font-size: 14px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.55;
    font-weight: 700;
    border-radius: 10px;
    background-color: #ad26ff;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;

}</string>
   </property>
   <property name="text">
    <string>2 место</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: #ffffff;
    font-size: 14px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.55;
    font-weight: 700;
    border-radius: 10px;
    background-color: #ad26ff;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;

}</string>
   </property>
   <property name="text">
    <string>3 место</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: #ffffff;
    font-size: 14px;
    font-family: 'Mulish', Arial, sans-serif;
    line-height: 1.55;
    font-weight: 700;
    border-radius: 10px;
    background-color: #ad26ff;
    background-position: center center;
    border-color: transparent;
    border-style: solid;
    transition: background-color 0.2s ease-in-out, color 0.2s ease-in-out, border-color 0.2s ease-in-out;

}</string>
   </property>
   <property name="text">
    <string>1 место</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
